test/jdk/sun/text/resources/LocaleDataTest.java

```diff
@@ -41,7 +41,7 @@
  *      8187946 8195478 8181157 8179071 8193552 8202026 8204269 8202537 8208746
  *      8209775 8221432 8227127 8230284 8231273 8233579 8234288 8250665 8255086
  *      8251317 8274658 8283277 8283805 8265315 8287868 8295564 8284840 8296715
- *      8301206 8303472 8317979 8306116 8174269
+ *      8301206 8303472 8317979 8306116 8174269 8333582
```

Comment:
Similar situation as TimeZoneNameTest.java? Looks like no other change than the bug id.

Reply:
There are changes in `LocaleData.cldr` data file, which is an input to this test case. Everytime the data file is modified, the bug id is recorded here.